chasm/lib/callback/invocation_tasks_test.go

```diff
@@ -0,0 +1,298 @@
+package callback_test
+
+import (
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/server/chasm"
+	"go.temporal.io/server/chasm/lib/callback"
+	callbackspb "go.temporal.io/server/chasm/lib/callback/gen/callbackpb/v1"
+	"go.temporal.io/server/common/backoff"
+	"go.temporal.io/server/common/dynamicconfig"
+	"go.temporal.io/server/service/history/queues"
+)
+
+// invocationTasksSuite extends callbackSuite to test InvocationTask execution.
+// These tests mirror HSM's TestProcessInvocationTaskNexus_Outcomes pattern.
+type invocationTasksSuite struct {
+	callbackSuite
+}
+
+func TestInvocationTasksSuite(t *testing.T) {
+	suite.Run(t, &invocationTasksSuite{})
+}
+
+func (s *invocationTasksSuite) SetupTest() {
+	s.callbackSuite.SetupTest()
+}
+
+// TestInvocationTask_SuccessfulNexusCallback tests a successful HTTP callback invocation.
+// Mirrors HSM test: TestProcessInvocationTaskNexus_Outcomes/success
+// TODO (seankane): This test is functionally a duplicate of TestExecuteInvocationTaskNexus_Outcomes/success
+// in executors_test.go until CHASM testing infrastructure matures to support verifying task generation
+// in integration tests. Once that's available this test will test the full CHASM stack integration.
+func (s *invocationTasksSuite) TestInvocationTask_SuccessfulNexusCallback() {
```

Comment:
What is this testing? We already have `TestExecuteInvocationTaskNexus_Outcomes` from the previous PR.